price-feed/src/frontend.rs

```diff
@@ -178,17 +185,18 @@ mod tests {
 	}
 
 	#[test]
-	fn test_get_usd_cent_price() {
+	fn test_get_normalized_price() {
+		let expected_exponent = Exponent(2);
 		[
-			((Price(0xCAFEBABE), Exponent(-2)), USDCentPrice(0xCAFEBABE)),
-			((Price(0xDEADBEEF), Exponent(2)), USDCentPrice(0xDEADBEEF * u64::pow(10, 2 + 2))),
-			((Price(1), Exponent(0)), USDCentPrice(u64::pow(10, 2))),
-			((Price(12), Exponent(-1)), USDCentPrice(12 * u64::pow(10, 1))),
-			((Price(454000), Exponent(-6)), USDCentPrice(45)),
+			((Price(0xCAFEBABE), Exponent(-2)), NormalizedPrice(0xCAFEBABE * u64::pow(10, 4))),
+			((Price(0xDEADBEEF), Exponent(2)), NormalizedPrice(0xDEADBEEF * u64::pow(10, 0))),
```

Comment:
can't we remove u64::pow() call here?

Reply:
I like it to be explicit w.r.t to the equation but changed it